app/src/main/java/com/my/journeyplanner/views/itineraryresults/ItineraryResultsCardviewListAdapter.kt

```diff
@@ -0,0 +1,59 @@
+package com.my.journeyplanner.views.itineraryresults
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.TextView
+import androidx.recyclerview.widget.DiffUtil
+import androidx.recyclerview.widget.ListAdapter
+import androidx.recyclerview.widget.RecyclerView
+import com.my.core.domain.JourneyPlannerResultDomainModel.Itinerary.Journey
+import com.my.journeyplanner.R
+import mu.KotlinLogging
+import org.threeten.bp.Duration
+import org.threeten.bp.format.DateTimeFormatter
+
+private val logger = KotlinLogging.logger {}
+
+class ItineraryResultsCardviewListAdapter(private val itemClickListener: (Journey) -> Unit) :
+    ListAdapter<Journey, ItineraryResultsCardviewListAdapter.MyViewHolder>(JourneyDiffCallback()) {
+
+    class MyViewHolder(var view: View) : RecyclerView.ViewHolder(view) {
+        val textViewJourney: TextView = view.findViewById(R.id.textViewJourney)
+    }
+
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int) =
+        MyViewHolder(
+            LayoutInflater.from(parent.context)
+                .inflate(R.layout.recycler_card_view_journey, parent, false)
```

Comment:
you don't have to call this layout recycler_, it doesn't really care where you put it

Reply:
Renamed to card_view_journey